packages/app/components/strategies/constants.tsx

```diff
@@ -93,7 +93,7 @@ export const STRATEGY_CATEGORIES: { [chainId: number]: ChainStrategies } = {
         {
           id: 3,
           buyToken: gnosisTokens.WETH,
-          daysAmount: 10,
+          daysAmount: 4,
```

Comment:
I know that 10 days of hourly trades is a lot. Any specific reason for choosing 4?

Reply:
It fits the current constraint of the maximum hours per hourly trade accepted in our custom timeframe input. Right now, we have a maximum of 96 hours, so 4 days * 24 hours = 96.

We can change any of these variables, but I made this change to fit the constraints explained above.